DigitalLearningSolutions.Web/Helpers/CentreCustomPromptHelper.cs

```diff
@@ -129,7 +132,21 @@ public List<CustomFieldViewModel> GetCustomFieldViewModelsForCentre(int centreId
             );
         }
 
-        public void ValidateCustomPrompts(EditDetailsFormData formData, int centreId, ModelStateDictionary modelState)
+        public void ValidateCustomPrompts(EditDelegateFormData formData, int centreId, ModelStateDictionary modelState)
+        {
+            ValidateCustomPrompts(
+                centreId,
+                formData.Answer1,
+                formData.Answer2,
+                formData.Answer3,
+                formData.Answer4,
+                formData.Answer5,
+                formData.Answer6,
+                modelState
+            );
+        }
+
+        public void ValidateCustomPrompts(MyAccountEditDetailsFormData formData, int centreId, ModelStateDictionary modelState)

```

Comment:
Can these two methods be consolidated as one that takes an EditDetailsFormData argument?

Reply:
Ah yes good spot, done.